src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Appearance/WindowBackdrop.cs

```diff
@@ -57,7 +57,8 @@ internal static bool ApplyBackdrop(System.Windows.Window window, WindowBackdropT
             return ApplyBackdrop(windowHandle, backdropType);
         }
 
-        window.Loaded += (sender, _) =>
+        RoutedEventHandler loadedHandler = null;
+        loadedHandler = (sender, _) =>
```

Comment:
Merge this into one. I mean why declare and assign seperately. Similarly for the other too. 
Marking this approved though as it is not blocking.

Reply:
we are unsubscribing the event inside the assignment, at which point the declaration is not yet done. Which will throw an error. Is there any other way we can do this?